### PR TITLE
Add lualine support && Add setting theme using command mode

### PR DESCRIPTION
**Pull request by donRehan**
Add support for nvim-lualine
Add ability to call theme from command mode by
typing :colorscheme neosolarized

### CODE REVIEW

**Comment by donRehan** [comment]
Hello svrana, 
I created this pull request to add lualine support as for me the theme didn't have it.
As well as allowing calling the theme using the colorscheme command from nvim.

If this goes along your idea of implementation of the theme would appreciate accepting the pull request.
Kind regards,
Alhussien

**Comment by svrana** [comment]
Hmm, the lualine code seems a bit out of scope since it's not setting vim config but just setting up a lualine specific data structure that than has to be required somewhere and input back into lualine. For those reasons, I think people looking to customize lualine should probably just carry this code themselves. 

fwiw, the screenshots in this repo use lualine's solarized-dark theme which I've found jive pretty well with what's going on here and honestly I cannot tell the difference between what you've got here and that one.

**Comment by donRehan** [comment]
image

Thank you for your quick response. I have failed to explain my reason as to why I did it in such format.

Regarding lualine code I  wrote , As you can see this is what happens when I simply call lualine without settings ; it looks like this. This happens with me on ( gnome-terminal , Konsole and st ) 
Hence is why I wrote the code that essentially returns the same effect of lualine dark_solarized. But ensures its more consistent across systems.

I can also add a protected call to return solarized_dark theme if it exists on the system if that fits your idea about your theme.
Let me know if you still prefer the current approach and hopefully I clarified my position.

**Comment by svrana** [comment]
But why call lualine without setting the theme to solarized_dark? i.e., require('lualline').setup { options = { theme = 'solarized_dark' } }

**Comment by donRehan** [comment]
To allow changing from one theme that changes lualine back to neosolarized fluent. without having to type into the lua config file.
Many themes have this by default it would be handy to have it in this lovely theme as well.

**Comment by svrana** [comment]
I see. Admittedly I don't change themes which is why this one doesn't do it well. Thanks for working on it :)

File: lua/neosolarized.lua [hunk 1]
```diff
@@ -127,8 +127,7 @@ function M.setup(opts)
 	Group.new("MoreMsg", colors.blue, colors.none, styles.NONE)
 	Group.new("ModeMsg", colors.blue, colors.none, styles.NONE)
 	Group.new("Question", colors.cyan, colors.none, styles.bold)
-	Group.new("WinSeparator", colors.base00, colors.none, styles.NONE)
-	Group.link("VertSplit", groups.WinSeparator)
+	Group.new("VertSplit", colors.base00, colors.none, styles.NONE)
```
Comment (svrana) [inline lua/neosolarized.lua:130]: Can you remove the changes to this file? Don't think they belong with the rest of the change.

Comment (donRehan) [reply lua/neosolarized.lua:130]: Yes its a mistake, I will remove this part as soon as I access my pc as I am outside now.

**Comment by donRehan** [comment]
I fixed the last commit mistake , we should be able to merge.
Please let me know once you review the files.

**Review by svrana — APPROVED** [review]
Thanks for the contribution @donRehan

**Comment by donRehan** [comment]
My pleasure ! @svrana

**Pull request merged**